pkg/stanza/operator/transformer/recombine/recombine_test.go

```diff
@@ -511,3 +556,39 @@ func TestTimeoutWhenAggregationKeepHappen(t *testing.T) {
 	}
 	require.NoError(t, recombine.Stop())
 }
+
+func TestSourceBatchDelete(t *testing.T) {
```

Comment:
Would you mind adding a test that triggers based on max batch size, and validates the logs that are emitted by the operator. (similar to your original example of 100 lines emitted as 50 + 50).

Reply:
yeah, a new test added.